simple/lint.go

```diff
@@ -1280,3 +1281,32 @@ func intLit(f *lint.File, expr ast.Expr) (int64, bool) {
 	val, _ := constant.Int64Val(tv.Value)
 	return val, true
 }
+
+func LintMakeLenCap(f *lint.File) {
+	fn := func(node ast.Node) bool {
+		call, ok := node.(*ast.CallExpr)
+		if !ok {
+			return true
+		}
+		if fn, ok := call.Fun.(*ast.Ident); !ok || fn.Name != "make" {
+			return true
```

Comment:
Add a `// FIXME`, saying that we should check whether `make` is indeed the built-in function.